### DISCUSSION TITLE
XCH Wallet Transfer Stuck

### DISCUSSION THREAD

**Discussion by VengefulSaint** [Chia-Network/chia-blockchain#17697] [Support]
7 days ago, I tried transferring some XCH to Crypto.com. That transaction is still pending. I've re-synced my wallet multiple times, tried deleting any unconfirmed transactions, and tried the claw back feature. None of this has helped fix the issue. What can I do?

**Comment by BrandtH22** [comment]
Hey @VengefulSaint , the chain has seen higher transaction volume recently so any transactions submitted without a fee can take time to process in the mempool. 

Feel free to jump into our discord server for more timely and thorough support (https://discord.gg/chia) but if the transaction has still not gone through I would delete unconfirmed transactions and send the funds with a fee. 

Side note - exchanges currently do not support the clawback feature so if you have a pending clawback you will need to claw it back on the sending wallet side then resubmit without a clawback.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
